x-pack/plugins/triggers_actions_ui/public/common/constants/aggregation_types.ts

```diff
@@ -8,7 +8,7 @@ import { AggregationType } from '../types';
 
 export enum AGGREGATION_TYPES {
   COUNT = 'count',
-  AVERAGE = 'avg',
+  AVERAGE = 'average',
```

Comment:
I just reviewed one of @pmuellr 's PRs where he did the opposite - changing `average` into `avg`.
Feels like we should keep this aligned? I'm not sure which is right though. 🤷‍♂